### ISSUE TITLE
add an impulse to rotate sentry to where player is pointing

### ISSUE THREAD

**Issue by Rickson-FOLD** [FortressOne/server-qwprogs#612]
This is EvilAnt's idea. I'm just putting it on here because I think it's a great idea.

It would eliminate the need to have the three different rotations (clock, counterclock and 180) and be simpler for the engy to execute. Additionally, it would be nice if, while building, the engy could set it and forget it.

(I still don't see where to mark this as a feature suggestion, bug, etc.)

**Comment by drzel** [comment]
Yeah I've always wanted the rotate sentry to give you the POV of the sentry, allow you to turn to set the sentry direction, and `+attack` to return to engy.
